src/ConversationList/ConversationListBox.vala

```diff
@@ -348,4 +349,15 @@ public class Mail.ConversationListBox : VirtualizingListBox {
     public void undo_expired () {
         move_handler.expire_undo ();
     }
+
+    public override bool query_tooltip (int x, int y, bool keyboard_tooltip, Gtk.Tooltip tooltip) {
+        unowned VirtualizingListBoxRow? row = get_row_at_y (y);
+        if (row != null) {
+            unowned ConversationItemModel model_item = (ConversationItemModel) row.model_item;
+            tooltip.set_markup (GLib.Markup.printf_escaped ("<b>%s</b>\n%s", model_item.from, model_item.subject));
+            return true;
+        }
+
+        return false;
+    }
```

Comment:
Since you set the `tooltip_markup` in `ConversationListItem` now, I guess we can completely remove this handler?

Reply:
It was on my local copy, I don't know what happened on GitHub side :man_shrugging:

Reply:
Had this already once in the past - I guess the code was re-added on my side somehow related to your force push. Deleted my local branch and pulled it again completely fresh. Looks good now 👍